libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -45,7 +53,23 @@ import Wire.API.UserMap
 
 type MaxUsersForListClientsBulk = 500
 
-type CheckUserExistsResponse = [EmptyResult 200, EmptyResult 404]
+type UserExistsVerb =
```

Comment:
There is a `UserExistVerb` in line 65 which seems to be exactly the same type. Remove duplicate?

Reply:
Yes, thanks for catching this. It was a mistake in a merge.